bindings/cs/MongoDB.Crypt/IKmsKeyId.cs

```diff
@@ -28,6 +31,14 @@ public interface IKmsKeyId
         /// The type of the KMS key.
         /// </value>
         KmsType KeyType { get; }
+
+        /// <summary>
+        /// Gets the alternate key names as byte arrays.
+        /// Each byte array describes an alternative key name via a BsonDocument in the following format:
+        ///  { "keyAltName" : [BSON UTF8 value] }
+        /// </summary>
+        /// <value>The alternate key names.</value>
+        IReadOnlyList<byte[]> AlternateKeyNames { get; }
```

Comment:
This should be a list of `string`.

Reply:
Technically the `AlternateKeyNames` are `AlternateKeyNameDocumentsAsByteBuffers` since each item in the list is a byte buffer that contains the binary representation of a `BsonDocument`: However, since the wrapper has no dependencies on any MongoDB.Bson, it relies on the caller to convert the `BsonDocument` to a byte array.